spec/index.html

```diff
@@ -4220,8 +4220,9 @@ <h3>CONSTRUCT</h3>
         </div>
         <section id="templatesWithBNodes">
           <h4>Templates with Blank Nodes</h4>
-          <p>A template can create an RDF graph containing blank nodes. The blank node labels are
-            scoped to the template for each solution. If the same label occurs twice in a template,
+          <p>A template can create an RDF graph containing blank nodes. The blank node labels inside the template are
+            scoped to the template for each solution, while blank node labels from query solutions are not scoped.
+            If the same label occurs twice in a template,
             then there will be one blank node created for each query solution, but there will be
             different blank nodes for triples generated by different query solutions.</p>
```

Comment:
As before but with s/identifier/label/: (there were other changes):
```suggestion
<p>A template can create an RDF graph containing blank nodes. 
            The blank node labels inside the template are scoped to the
            template for each solution, while blank nodes from query solutions
            are not scoped.
            If the same label occurs twice in a template, every occurrence
            is replaced by the same blank node which is created for each
            query solution, and there will be different blank nodes for triples
            generated by different query solutions.
          </p>
```

Reply:
I'm a bit confused. I thought we were going to use "identifier" instead of "label".
Or do you want to omit this change from this PR, and move it to #69?

Reply:
Sorry for the confusion.

In https://github.com/w3c/sparql-query/pull/65#discussion_r1185932111 I understood it that you wanted to keep the label/identifier thing apart from this PR. The original suggested change was "resolved" and not incorporated - that's fine.

At this point, I'd prefer to this PR with the two "labels" version of the suggestions as soon as convenient. Then I'll update the #69 for all the label/identifier changes in one PR for that change.

I'm keen to get #69 merged because it touches many sections. I hope to address the technical errata next. It will be easier to work on errata with a more stable underlying document.

Reply:
Ah, apologies for the confusion, I meant to merge your suggestion, but apparently didn't...

> At this point, I'd prefer to this PR with the two "labels" version of the suggestions as soon as convenient. Then I'll update the https://github.com/w3c/sparql-query/pull/69 for all the label/identifier changes in one PR for that change.

Sounds good to me!